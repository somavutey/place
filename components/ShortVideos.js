import ReactPlayer from "react-player";
const ShortVideos = ({ video_url }) => {
  return (
    <div>
      <ReactPlayer
        controls
        width="300px"
        height="60vh"
        url={video_url}
      ></ReactPlayer>
    </div>
  );
};

export default ShortVideos;
