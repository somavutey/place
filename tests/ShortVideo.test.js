import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import ShortVideos from "./../components/ShortVideos";

it("ShortVideos", () => {
  const wrapper = shallow(<ShortVideos />);
  expect(toJson(wrapper)).toMatchSnapshot();
});