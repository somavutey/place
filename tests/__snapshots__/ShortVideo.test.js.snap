// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShortVideos 1`] = `
<div>
  <ReactPlayer
    config={
      Object {
        "dailymotion": Object {
          "params": Object {
            "api": 1,
            "endscreen-enable": false,
          },
        },
        "facebook": Object {
          "appId": "1309697205772819",
          "attributes": Object {},
          "playerId": null,
          "version": "v3.3",
        },
        "file": Object {
          "attributes": Object {},
          "dashVersion": "3.1.3",
          "flvVersion": "1.5.0",
          "forceAudio": false,
          "forceDASH": false,
          "forceFLV": false,
          "forceHLS": false,
          "forceVideo": false,
          "hlsOptions": Object {},
          "hlsVersion": "0.14.16",
          "tracks": Array [],
        },
        "mixcloud": Object {
          "options": Object {
            "hide_cover": 1,
          },
        },
        "soundcloud": Object {
          "options": Object {
            "buying": false,
            "download": false,
            "liking": false,
            "sharing": false,
            "show_comments": false,
            "show_playcount": false,
            "visual": true,
          },
        },
        "twitch": Object {
          "options": Object {},
          "playerId": null,
        },
        "vidyard": Object {
          "options": Object {},
        },
        "vimeo": Object {
          "playerOptions": Object {
            "autopause": false,
            "byline": false,
            "portrait": false,
            "title": false,
          },
        },
        "wistia": Object {
          "customControls": null,
          "options": Object {},
          "playerId": null,
        },
        "youtube": Object {
          "embedOptions": Object {},
          "onUnstarted": [Function],
          "playerVars": Object {
            "iv_load_policy": 3,
            "modestbranding": 1,
            "playsinline": 1,
            "rel": 0,
            "showinfo": 0,
          },
        },
      }
    }
    controls={true}
    fallback={null}
    height="60vh"
    light={false}
    loop={false}
    muted={false}
    onBuffer={[Function]}
    onBufferEnd={[Function]}
    onClickPreview={[Function]}
    onDisablePIP={[Function]}
    onDuration={[Function]}
    onEnablePIP={[Function]}
    onEnded={[Function]}
    onError={[Function]}
    onPause={[Function]}
    onPlay={[Function]}
    onProgress={[Function]}
    onReady={[Function]}
    onSeek={[Function]}
    onStart={[Function]}
    pip={false}
    playbackRate={1}
    playing={false}
    playsinline={false}
    previewTabIndex={0}
    progressInterval={1000}
    stopOnUnmount={true}
    style={Object {}}
    volume={null}
    width="300px"
    wrapper="div"
  />
</div>
`;
